### PR TITLE
chore(docs): Update using-graphql-playground

### PR DESCRIPTION
**Pull request by frankievalentine**
Added missing step to require dotenv package, misc formatting changes

### CODE REVIEW

**Comment by pieh** [comment]
Hi @frankievalentine! Thanks for PR!

Few notes:
You don't need `dotenv` - running `GATSBY_GRAPHQL_IDE=playground gatsby develop` is enough on most most platforms (not counting Windows), but if you apply it to your scripts in package.json (as currently documented) and run `npm/yarn run develop` - it will work everywhere. 

It's either `dotenv` approach or the steps documented currently. and this changes is making it a bit more confusing because it seems like you need to do all the steps

**Comment by frankievalentine** [comment]
> Hi @frankievalentine! Thanks for PR!
> 
> Few notes:
> You don't need `dotenv` - running `GATSBY_GRAPHQL_IDE=playground gatsby develop` is enough on most most platforms (not counting Windows), but if you apply it to your scripts in package.json (as currently documented) and run `npm/yarn run develop` - it will work everywhere.
> 
> It's either `dotenv` approach or the steps documented currently. and this changes is making it a bit more confusing because it seems like you need to do all the steps

Hi @pieh 
Right so I guess that is where there is some confusion in this doc. Typically, in development, it is known to utilize `gatsby develop` instead of `npm/yarn run develop` so I guess that's where the documentation doesn't state specifically to use 'npm/yarn run develop' instead of `gatsby develop` to utilize Playground.

Would you advise I edit the PR to include both .env methods?

**Comment by pieh** [comment]
> Would you advise I edit the PR to include both .env methods?

I think, this way we would end up having to duplicate those methods in every docs page that uses environment variables. Ideally we could just say to use `GATSBY_GRAPHQL_IDE=playground` variable and link to section in https://www.gatsbyjs.org/docs/environment-variables/ that document those 2 methods so we don't need to duplicate it everywhere

**Comment by frankievalentine** [comment]
> > Would you advise I edit the PR to include both .env methods?
> 
> I think, this way we would end up having to duplicate those methods in every docs page that uses environment variables. Ideally we could just say to use `GATSBY_GRAPHQL_IDE=playground` variable and link to section in https://www.gatsbyjs.org/docs/environment-variables/ that document those 2 methods so we don't need to duplicate it everywhere

In just the case of using Playground, I don't think this causes duplication. How I see it currently, yes, the easiest method is to add the script to your package.json file and use `npm/yarn run develop`. Which, I believe this doc for this PR should include that you use `npm/yarn run develop` versus `gatsby develop`.

But, if you still wish to use `gatsby develop` you would add the 'dotenv' package to your gatsby-config.js file and add the environment variable file.

I think there is a significant difference, not enough to change the process as a whole, but one that other developers should know about.

**Comment by frankievalentine** [comment]
I have updated the README to include the correct instructions for utilizing GraphQL Playground with `npm run develop` and `gatsby develop`.

File: docs/docs/using-graphql-playground.md [hunk 1]
```diff
@@ -4,20 +4,22 @@ title: Using the GraphQL Playground
...
-## What is The Prisma Playground?
+## What is Prisma's GraphQL Playground?
 
-The Playground is a way for you to interact with the data your sources and plugins add as schemas. You will be interacting with this data a lot and the Playground will help you greatly with exploring this data.
+GraphQL Playground is a way for you to interact with the data your sources and plugins add as schemas. You will be interacting with this data a lot and the Playground will help you greatly with exploring this data.
 
 ## Accessing the Playground
 
-As this is an experimental feature, you need to access it by adding an environmental variable, you need to add `GATSBY_GRAPHQL_IDE` to your `develop` script in your `package.json`, like this:
+To access GraphQL Playground add `GATSBY_GRAPHQL_IDE` to your `develop` script in your `package.json`, like this:
```
Comment (KyleAMathews) [inline docs/docs/using-graphql-playground.md]: Could you add back the note about this being an experimental feature? That's important as we may remove it in a future update so people shouldn't expect this to stay around.

Comment (frankievalentine) [reply docs/docs/using-graphql-playground.md]: Sent a new commit with the note included again!

**Pull request merged**